### PR TITLE
Updates a few genetics icons and adds some new ones

### PR DESCRIPTION
**Pull request by WALPVRGIS**
## About the PR 

- Updates some genetic power icons to be a lil less blurry and to use resprite humans when needed
- Changes a few icons to be more readable from a design standpoint
- Adds some icons for genes that I see in the booth often enough (such as dermal glitter, cranial keratin formation etc)

geneiconexamples
(Just an example, check the icon diffs for the full list of updated icons)

Also this does not update the icons seen in the TGUI genetics interface (see Mordent's comment below)

## Why's this needed? 
Couple of the icons looked a little dated, and a few were a bit unclear as to what they would do, especially for newer players messing with gene powers for the first time. Would be nice to have icons for all mutations someday, maybe!

### CODE REVIEW

**Comment by mordent-goonstation** [comment]
I'm honestly considering stripping out the genetics icons from the GeneTek - *possibly* going the opposite way and making them much more visible. Right now it's in a weird middle-ground of sending a good chunk of data that is used in a way that doesn't actually really help disambiguate anything, so is just visual clutter.

**Comment by WALPVRGIS** [comment]
> 
> 
> I'm honestly considering stripping out the genetics icons from the GeneTek - _possibly_ going the opposite way and making them much more visible, but that's a longer term/bigger thing. Right now it's in a weird middle-ground of sending a good chunk of data that is used in a way that doesn't actually really help disambiguate anything, so is just visual clutter.
> 
> Basically: no need to hold this PR up because of the GeneTek UI.

Thank you for the heads up! :)

File: code/modules/medical/genetics/bioEffects/useless.dm [hunk 1]
```diff
@@ -288,6 +293,7 @@
 	name = "Bee Production"
 	desc = "Encourages growth of bees in the subject's body."
 	id = "drunk_bee"
+	icon_state = "unknown"
```
Comment (mordent-goonstation) [inline code/modules/medical/genetics/bioEffects/useless.dm:296]: This is the behavior on the base bio effect so you do not need to specify here. Ditto for the other `"unknown"` ones added.

**Review by mordent-goonstation — CHANGES_REQUESTED** [review]

File: code/modules/medical/genetics/bioEffects/useless.dm (hunk 1 above)
Comment (WALPVRGIS) [reply code/modules/medical/genetics/bioEffects/useless.dm:296]: Ah, I see! I'll edit that shortly, ty! :D

**Pull request merged**